sdk/ai/Azure.AI.Agents/tests/Azure.AI.Agents.Tests.csproj

```diff
@@ -0,0 +1,46 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFrameworks>$(RequiredTargetFrameworks)</TargetFrameworks>
+
+    <!--Ignore XML doc comments on test types and members-->
+    <NoWarn>$(NoWarn);CS1591</NoWarn>
+
+    <!-- Package downgrade (BCL) -->
+    <NoWarn>$(NoWarn);NU1605</NoWarn>
+
+    <!-- Temporary fix to include latest changes from Projects. -->
+    <NoWarn>$(NoWarn);NU1105</NoWarn>
+
+    <!-- OPENAI001 - Ignore experimental warnings-->
+    <NoWarn>$(NoWarn);OPENAI001;SCME0001;</NoWarn>
+
+    <LangVersion>latest</LangVersion>
+  </PropertyGroup>
+  <ItemGroup>
+    <ProjectReference Include="..\src\Azure.AI.Agents.csproj" />
+    <ProjectReference Include="..\..\Azure.AI.Projects\src\Azure.AI.Projects.csproj" />
+  </ItemGroup>
+  <ItemGroup>
+    <PackageReference Include="Azure.Identity" />
+    <PackageReference Include="Microsoft.Bcl.AsyncInterfaces" VersionOverride="9.0.0" />
+    <PackageReference Include="NUnit" VersionOverride="4.4.0" />
+    <PackageReference Include="NUnit3TestAdapter" VersionOverride="5.1.0"/>
+    <PackageReference Include="Microsoft.NET.Test.Sdk" />
+    <PackageReference Include="Moq" />
+    <PackageReference Include="OpenTelemetry.Exporter.Console" />
+    <PackageReference Include="System.Linq.Async" />
+		<PackageReference Include="Microsoft.ClientModel.TestFramework" VersionOverride="1.0.0-alpha.20251023.1"/>
```

Comment:
@m-redding:  This feels like the wrong approach to including for a project in the SDK repo.   Should we be taking a project ref like we do with the Azure test framework?   If not, we should add this to the central versions rather than local "forever" overrides.

Reply:
Either way would be fine, I will add it to the central packages references, I think that would be preferable to using project references

Reply:
I've updated the PR to add it to what I believe is the appropriate location in the central references -- 100% defer to the *more* appropriate place if I got it wrong!